modules/optics/arrow-optics/src/main/kotlin/arrow/optics/predef.kt

```diff
@@ -5,8 +5,7 @@ import arrow.core.Option
 import arrow.typeclasses.Const
 import arrow.typeclasses.Monoid
 
-@PublishedApi
-internal object AndMonoid : Monoid<Boolean> {
+object AndMonoid : Monoid<Boolean> {
```

Comment:
Why is this not internal anymore?

Reply:
I guess I was wrong - IDE shows error, but it compiles fine.

Reply:
Never seen that happen before hahaha

Reply:
https://youtrack.jetbrains.com/issue/KT-23668 - it's propably that